docs/db/changelog/changesets/powerauth-java-server/1.8.x/db.changelog-version.xml

```diff
@@ -5,5 +5,6 @@
                    https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
 
     <include file="20240424-index-expire-status.xml" relativeToChangelogFile="true" />
+    <include file="20240517-fido2-authenticator-transports.xml" relativeToChangelogFile="true" />
```

Comment:
Do we want to backport this? I am asking because of the changelog version.

Reply:
We most likely will not need to backport this feature.